parm/use_cases/model_applications/short_range/EnsembleStat_fcstHRRRE_obsHRRRE_Sfc_MultiField.conf

```
@@ -179,22 +185,47 @@ ENSEMBLE_STAT_OUTPUT_FLAG_ORANK = BOTH
 ENSEMBLE_STAT_OUTPUT_FLAG_SSVAR = BOTH
 ENSEMBLE_STAT_OUTPUT_FLAG_RELP = BOTH
 
-ENSEMBLE_STAT_ENSEMBLE_FLAG_LATLON = TRUE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_MEAN = TRUE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_STDEV = TRUE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_MINUS = FALSE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_PLUS = FALSE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_MIN = FALSE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_MAX = FALSE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_RANGE = FALSE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_VLD_COUNT = TRUE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_FREQUENCY = TRUE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_NEP = FALSE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_NMEP = FALSE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_RANK = FALSE
-ENSEMBLE_STAT_ENSEMBLE_FLAG_WEIGHT = FALSE
+
+ENSEMBLE_STAT_NC_ORANK_FLAG_LATLON = TRUE
+ENSEMBLE_STAT_NC_ORANK_FLAG_MEAN = TRUE
+ENSEMBLE_STAT_NC_ORANK_FLAG_RANK = FALSE
+ENSEMBLE_STAT_NC_ORANK_FLAG_PIT = TRUE
```

```suggestion
ENSEMBLE_STAT_NC_ORANK_FLAG_PIT = FALSE
```
Previously, ens.rank controlled the writing of RANK and PIT. When RANK is false, also making PIT FALSE is more consistent with previous outputs.